sycl/test-e2e/Matrix/joint_matrix_int8_colmajorA_colmajorB.cpp

```diff
@@ -15,9 +15,9 @@
 // RUN: %{run} %t.out
 
 // This tests support of col major layout for matrix B which does transpose and
-// then VNNI transform. This is currently only available on AMX
+// then VNNI transform. This is currently only available on AMX and PVC
 
-// XFAIL: gpu
+// XFAIL: gpu-intel-dg2 || arch-intel_gpu_bmg_g21
```

Comment:
already changed like that:
https://github.com/intel/llvm/blob/8d34f2aa1ae6e8e5509c5f44f8f3cf24e08971cf/sycl/test-e2e/Matrix/joint_matrix_int8_colmajorA_colmajorB.cpp#L20